docs/architecture/psa-migration/outcome-analysis.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/sh
```

Comment:
If this is going to become a CI fixture, I'd prefer to use Python, which is better at complex stuff and which more people know. Also, it could use a name that's more distinct from `analyze_outcomes.py`.